pkg/sdk/testint/tasks_gen_integration_test.go

```diff
@@ -422,8 +429,8 @@ func TestInt_Tasks(t *testing.T) {
 			WithAllowOverlappingExecution(true).
 			WithUserTaskTimeoutMs(10).
 			WithSessionParameters(sdk.SessionParameters{
-				// TODO(SNOW-1348116 - next prs): fill and assert parameters
-				Autocommit: sdk.Bool(true),
+				Autocommit:         sdk.Bool(false),
+				AbortDetachedQuery: sdk.Bool(true),
```

Comment:
Are we planning to test all of the task parameters here?

Reply:
Added